### PR TITLE
NTR: Stotting

### LINKED ISSUE #108: NTR: stotting
For new term requests, please provide the following information:

## Preferred term label

stotting

## Synonyms

pronking
pronging

## Textual definition

a pursuit deterrence behavior of quadripeds in which they spring into the air, lifting all four feet off the ground simultaneously with the legs held in a relatively stiff position.
## Suggested parent term

predator avoidance behavior

## Attribution

http://orcid.org/0000-0002-2908-332

### PR DESCRIPTION
**Pull request by DitchingIt**
Fixes #108

### CODE REVIEW

**Review by matentzn — COMMENTED** [review]
Look at #171 for comments on the form of the reference.

**Comment by DitchingIt** [comment]
DOI added

File: src/ontology/nbo-edit.owl [hunk 1]
```diff
@@ -9944,6 +9947,18 @@ AnnotationAssertion(oboInOwl:hasOBONamespace obo:NBO_0050002 "behavior_ontology"
 AnnotationAssertion(rdfs:label obo:NBO_0050002 "circling behavior")
 SubClassOf(obo:NBO_0050002 obo:NBO_0000013)
 
+# Class: obo:NBO_0070004 (stotting)
+
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "DOI 10.1016/S0003-3472(86)80051-3") obo:IAO_0000115 obo:NBO_0070004 "Display that involves taking all four legs off the ground simultaneously and holding the legs stiff and straight.")
```
Comment (matentzn) [inline src/ontology/nbo-edit.owl:9952]:
```suggestion
AnnotationAssertion(Annotation(oboInOwl:hasDbXref "DOI:10.1016/S0003-3472(86)80051-3") obo:IAO_0000115 obo:NBO_0070004 "Display that involves taking all four legs off the ground simultaneously and holding the legs stiff and straight.")
```

**Review by matentzn — APPROVED** [review]

File: src/ontology/nbo-edit.owl [hunk 2]
```diff
@@ -9949,6 +9952,18 @@ AnnotationAssertion(oboInOwl:hasOBONamespace obo:NBO_0050002 "behavior_ontology"
 AnnotationAssertion(rdfs:label obo:NBO_0050002 "circling behavior")
 SubClassOf(obo:NBO_0050002 obo:NBO_0000013)
 
+# Class: obo:NBO_0070004 (stotting)
+
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "DOI:10.1016/S0003-3472(86)80051-3") obo:IAO_0000115 obo:NBO_0070004 "Display that involves taking all four legs off the ground simultaneously and holding the legs stiff and straight.")
+AnnotationAssertion(dc:contributor obo:NBO_0070004 <http://orcid.org/0000-0002-2908-332>)
```
Comment (matentzn) [inline src/ontology/nbo-edit.owl:9958]:
```suggestion
AnnotationAssertion(terms:contributor obo:NBO_0070004 <http://orcid.org/0000-0002-2908-332>)
```

Comment (matentzn) [reply src/ontology/nbo-edit.owl:9958]: Only use terms:contributor

**Review by matentzn — APPROVED** [review]
Careful also with http vs https when using orcids

**Pull request merged**